markdown/publications.md

```diff
@@ -3,11 +3,21 @@ Please [make a pull-request](https://github.com/nf-core/nf-co.re/blob/master/mar
 
 # The nf-core project
 
-Preprint for the main nf-core paper, describing the community and framework:
+Main publication for the main nf-core paper, describing the community and framework:
+
+<!-- nf-core-altmetric -->
+> **The nf-core framework for community-curated bioinformatics pipelines**
+>
+> Philip Ewels, Alexander Peltzer, Sven Fillinger, Harshil Patel, Johannes Alneberg, Andreas Wilm, Maxime Ulysse Garcia, Paolo Di Tommaso & Sven Nahnsen
+>
+> [_Nat Biotechnolgy_ (2020)](https://doi.org/10.1038/s41587-020-0439-x);
```

Comment:
Wrong spelling!! :eyes: Need to add this to the pipeline template. I'll do that tomorrow :smile:

Reply:
Oops! It’s because the website citation is `Nature Biotechnolg` which I thought sounded a bit crap. But didn’t add back enough letters 🤦🏻‍♂️